docs/user-guide/custom-themes.md

```diff
@@ -6,12 +6,15 @@ A guide to creating and distributing custom themes.
 
 !!! Note
 
-    If you are looking for third party themes, they are listed in the MkDocs
```

Comment:
The indentation is intentional here. If you notice, the previous line contains the three exclamation points followed by an Admonition type (`!!! Note`). This is using the [Admonition Extension](https://pythonhosted.org/Markdown/extensions/admonition.html) syntax and the indented paragraph is the body of the note. Your change breaks it. If it helps, the Admonition Extension is enabled in our [config file](https://github.com/mkdocs/mkdocs/blob/master/mkdocs.yml#L27).